openedx/edx-ora2
# feat: make studio UI aware of override

**TL;DR -** Make ORA studio UI aware of flexible course-wide override

JIRA:  https://2u-internal.atlassian.net/browse/AU-1322

**What changed?**

- Add override to studio context
- If it's enabled, disable the select box for flexbile grading
- If it's enabled, show a message that the setting is enabled.

**Developer Checklist**


**Testing Instructions**

- Navigate to an ORA with a peer step in a course with the override turned off. The flexible setting should be active and there should be no message about the override.
- Turn the override on
- The input in the ORA settings should now be disabled and there should be a warning message.

![Screenshot 2023-07-14 at 11 34 31 AM](https://github.com/openedx/edx-ora2/assets/1639231/621e4ac6-8364-46df-80c8-f9b1318869e0)


**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @openedx/content-aurora
